distribution/docs/src/main/resources/content/_configuring/wss-using-standalone-servers.adoc

```diff
@@ -1,5 +1,5 @@
 :title: Configuring WSS Using Standalone Servers
```

Comment:
❓ This page used to be the CAS configuration guide (see [here](http://artifacts.codice.org/service/local/repositories/releases/content/ddf/docs/2.8.0/docs-2.8.0-Managing.html#_configure_wss_using_standalone_servers)) but doesn't add anything anymore. Can we just remove it altogether?

Reply:
Sure.